tests/unit_tests/test_wave_02.py

```diff
@@ -34,7 +34,9 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("electronics")
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
+    assert(items == [])
+    assert(vendor.inventory == [item_a, item_b, item_c])
```

Comment:
Great asserts! The second assert isn't necessary since the function doesn't change what's in the inventory.